spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/HyperConformance.scala

```diff
@@ -128,6 +128,7 @@ object HyperConformance extends StreamTransformerFactory with HyperConformanceAt
       datasetName = conf.getString(datasetNameKey),
       datasetVersion = conf.getInt(datasetVersionKey),
       reportDate = new SimpleDateFormat(ReportDateFormat).format(new Date()),
+      reportVersion = Option(getReportVersion(conf)),
```

Comment:
```suggestion
      reportVersion = Option(getReportVersion(conf)),
      performanceMetricsFile = None,
      folderPrefix = None,
      persistStorageLevel = None,
```
Perhaps add the other fields that were there originally (I know they equal to default)